cmake/developer_package/compile_flags/sanitizer.cmake

```
@@ -17,12 +17,20 @@ if (ENABLE_SANITIZER)
             "https://github.com/openvinotoolkit/openvino/wiki/AddressSanitizer-and-LeakSanitizer")
         endif()
     elseif(CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG)
-        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fsanitize=address")
+        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fsanitize=address -fsanitize-blacklist=${PROJECT_SOURCE_DIR}/tests/asan/ignore.txt")
```

```suggestion
        set(SANITIZER_COMPILER_FLAGS "${OpenVINO_SOURCE_DIR} -fsanitize=address -fsanitize-blacklist=${PROJECT_SOURCE_DIR}/tests/asan/ignore.txt")
```

For targets that are build on top of existed OV it produces wrong path (e.g. for fuzzer tests it will be `openvino/tests/fuzz/tests/asan/ignore.txt`)

---

Merged the changes from #23330